src/app/pages/clients/clients.component.html

```diff
@@ -0,0 +1,61 @@
+<div>
+  <div class="row">
+    <div class="col-md-8">
+      <div class="row">
+        <div class="col-md-6" *ngFor="let client of clients">
+          <div class="card listview">
+            <div class="row no-gutters">
+              <div class="col">
+                <div class="card-block px-2">
+                  <h4>
+                    <b class="card-title"> {{ client.client_name }}</b>
+                  </h4>
+
+                  <ul class="list-unstyled fa-ul">
+                    <li>
+                      <i class="fa fa-street-view"></i>
+                      <span
+                        >{{ client.address.Street }}, {{ client.address.City }},
+                        {{ client.address.State }}
+                      </span>
+                    </li>
+                    <li>
+                      <i class="fa fa-phone"></i>
+                      <span>{{ client.phone }}</span>
+                    </li>
+                    <li>
+                      <i class="fa fa-envelope"></i
+                      ><span>{{ client.email }}</span>
+                    </li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+            <div class="card-text">
+              <p>
+                <b class="client-reviews">Reviews</b>
+
+                <span class="row">
+                  <a href="/client/:clientId/review" class="card-link col"
+                    >Write a review</a
+                  >
+                  <a href="/client/:clientId/review" class="card-link col"
```

Comment:
You can also change this to dynamic URL to avoid coming back to edit URLs

Reply:
this is not part of her task as this will require her to test the navigation which is another implementation to her task